build.sbt

```diff
@@ -298,7 +296,7 @@ lazy val docs = http4sProject("docs")
         case _ => Seq.empty
       }
     },
-    scalacOptions in (Compile,doc) -= "-Ywarn-unused-import",
+    scalacOptions in (Compile,doc) -= "-Ywarn-unused:imports",
```

Comment:
I predict we're going to need a lot more than this when Travis gets rolling.

Reply:
The `sbt-tpolecat` plugin has this option: `scalacOptions.in(Tut) ~= filterConsoleScalacOptions`, which might be useful here?

I guess it just filters out the things mentioned in Rob's blog post.

Reply:
I like how sbt-tpolecat does the Right Thing in other contexts.  When we figure out the exact Right Thing here, I think we should consider a PR to that plugin.